lib/helpers.js

```diff
@@ -115,6 +116,14 @@ export function firstImplementer(...targets) {
   });
 }
 
+function isRoot(dir) {
+  return path.resolve(dir, '..') === dir;
```

Comment:
:+1:

Reply:
:heart: this just gave me an idea for one of the bugs I've been trying to fix!